### DISCUSSION TITLE
Config Option to Generate Zero-Argument Constructor Only?

### DISCUSSION THREAD

**Discussion by toinetoine** [kobylynskyi/graphql-java-codegen#975] [Q&A] [answered]
Is there anyway to generate **only** a zero-argument constructor? I couldn't see anything in the configuration documentation.

Wondering if there's a way so instead of generating **both** a zero argument constructor and all-argument constructor that [graphql-java-codegen](https://github.com/kobylynskyi/graphql-java-codegen) generates by default, could make it generate **only** the zero argument constructor (have the generated build method use getters instead). 

Reason is I'm breaching java limit of 255 params due to having extremely large schema types definitions (out of my control) of 500+ fields.

Thanks.

**Comment by kobylynskyi** [comment, marked as answer]
Thanks for your inquiry.
Currently it is not possible.
